docs/data-sources/aws_bucket_policy.md

```diff
@@ -75,6 +75,7 @@ resource "aws_s3_bucket_policy" "ds" {
 ## Argument Reference
 
 * `bucket` - (Required) AWS S3 Bucket name for which to generate the policy document.
+* `aws_partition` - (Optional) AWS partition. The options are `aws`, `aws-cn`, or `aws-us-gov`.
```

Comment:
add that `aws` is the default value for this option? And do it in all documents?

Reply:
Done 🙌